OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 12

# Description
Lint fix for Reports component, this is phase 12. This pull request only fixes two files. PeopleReport.jsx is a big file, including 700 lines of code. I think I should create another pull request for that.
## Files changed
src/components/Reports/PeopleReport/index.js
src/components/Reports/PeopleReport/selectors.js

eslint test:
![CleanShot 2023-10-05 at 02 05 09@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/f84ea07a-e4e2-48e7-8aac-aceb136fa924)


# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/PeopleReport/selectors.js`
- `npx eslint src/components/Reports/PeopleReport/selectors.js`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
